API_DOCUMENTATION.md

```
@@ -0,0 +1,873 @@
+./gradlew packageDebugPlugin
+```
+
+2. Deploy plugin files to host app assets or download location
+
+3. Initialize plugin manager in host app and load plugins as needed
+
+---
+
+## Best Practices
+
+### 1. Resource Management
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)